### ISSUE TITLE
Error in line 5

### ISSUE THREAD

**Issue by huphlungpu** [derfabsSL/Splinterbuyer#13]
when i run the bot from cmd in windows it says

C:\winold\personal\Splinterbuyer>python splinterbuyer.py
Traceback (most recent call last):
  File "C:\winold\personal\Splinterbuyer\splinterbuyer.py", line 5, in <module>
    from beem.blockchain import Blockchain
ModuleNotFoundError: No module named 'beem'

if i got to install it through pip it goes through the whole process but it says this at the end

      building '_scrypt' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for scrypt
Failed to build scrypt
ERROR: Could not build wheels for scrypt, which is required to install pyproject.toml-based projects

But i went to MS Visual and downloaded and installed.  So I'm not sure what the issue and where to go to solve it.

TIA for any help

**Comment by derfabsSL** [comment]
Try this pls and see if the error persists

```
pip install -r requirements.txt
```

**Comment by derfabsSL** [comment]
Also pls tell me your python version
```
python --version
```

**Comment by huphlungpu** [comment]
i got it all solved.  First of all just download the visual studio build tools and select desktop components and it auto selects latest files.  download and install.  also ran into more errors. PIP command not recognized.  I went and reinstalled python and that solved the issue.  it put python in path and then ran pip command.  Then the last issue is that it said scrypt wheel error.  couldn't build it.  I had to download and install openssl.  hope that helps someone in the future
